Gateway/Command/Capture.php

```diff
@@ -175,6 +175,7 @@ public function execute(array $commandSubject)
                     PriceCurrencyInterface::DEFAULT_PRECISION,
                     $order->getStoreId()
                 );
+                $formattedDiscountAmount = preg_replace('/\xc2\xa0/', ' ', $formattedDiscountAmount);
```

Comment:
There are many control characters that are invisible that should probably be removed. Wouldn't something like this take care of it all?

```suggestion
                $formattedDiscountAmount = preg_replace('/[\x00-\x1F\x7F]/', ' ', $formattedDiscountAmount);
```